src/Common/constants.tsx

```diff
@@ -224,6 +223,7 @@ export const ADMITTED_TO = [
   "ICU with Non Invasive Ventilator",
   "ICU with Oxygen Support",
   "ICU with Invasive Ventilator",
+  "BED WITH OXYGEN SUPPORT",
```

Comment:
Do we have this bed type in backend?

Reply:
I added based on issue  #871 
But need to check the backend

Reply:
@vigneshhari can you confirm this?

Reply:
will confirm soon